.github/workflows/ci.yml

```diff
@@ -81,6 +81,9 @@ jobs:
       run: git submodule update --init --depth 1 fuzz/corpora
     - name: localegen
       run: sudo locale-gen tr_TR.UTF-8
+    - name: fipsvendor
+      # Make one fips build use a customized FIPS vendor
+      run: echo "FIPSVENDOR=CI" >> VERSION.dat
```

Comment:
Could we also add a check (executing `openssl list -providers`) that the custom vendor name is there?